### DISCUSSION TITLE
2학기를 앞두고 방학을 돌아보다

### DISCUSSION THREAD

**Discussion by KwonNaeHyeon** [BDD-CLUB/challenge-every-week-posting#166] [5. 회고]
안녕하세요. 9월 6일 자로 글쓰기 챌린지에 참여하게 되었습니다. 
글쓰기에는 실력이 없는 편이지만, 열심히 글쓰기에 참여하여 글 조직력을 올릴 수 있도록 노력하겠습니다. 
2학기 첫 주차를 보내며 방학에 대한 회고록을 작성해 보았으며, 글을 쓰며 생각했던 다짐을 지키며 2학기를 열심히 보내겠습니다.
[2학기를 앞두고 방학을 돌아보다](https://lateral-value-9a9.notion.site/2-cf87e1352a494f94b69156e9ef1d3971?pvs=4)

**Comment by shkisme** [comment]
화이팅해봅시다~! 👍👍
